File: MD/ui/src/emails/invitation-link.ts
export const InvitationLinkEmail = (teamName: string, inviteUrl: string) => `
  <h2>You are invited to join team: ${teamName}</h2>
  <p>Click the link below to accept your invitation:</p>
  <p><a href="${inviteUrl}">${inviteUrl}</a></p>
`;

File: rust-coolify/crates/notify/src/templates/invitation_link.rs
// completed file_0850_invitation_link
// Coolify mənbəsi: app/Notifications/TransactionalEmails/InvitationLink.php

pub struct InvitationLinkTransactionalEmail;

impl InvitationLinkTransactionalEmail {
    pub fn format_message(team_name: &str, invite_link: &str) -> String {
        format!(
            "<h2>You Have Been Invited!</h2><p>You were invited to join team <b>{}</b> on Coolify.</p><p><a href=\"{}\">Accept Invitation</a></p>",
            team_name, invite_link
        )
    }
}
